deployment/ansible/roles/nih-wayfinding.otp-data/tasks/main.yml

```diff
@@ -3,9 +3,9 @@
 - name: Create Directory for OTP data
   file: path={{ otp_data_dir }} state=directory owner={{ansible_ssh_user}} mode=0775
 
-- name: Copy OSM Data
+- name: Copy OTP Data
   synchronize: src={{ item }} dest="{{ otp_data_dir }}/{{ item }}"
-  with_items: otp_osm_source_files
+  with_items: otp_osm_source_files.split()
```

Comment:
Not sure why it's necessary to build/split the filenames string to just copy over directory contents.  Can ansible [copy](http://docs.ansible.com/copy_module.html) not handle this?